include/zephyr/net/wifi.h

```diff
@@ -46,8 +46,12 @@ enum wifi_security_type {
 	WIFI_SECURITY_TYPE_PSK,
 	/** WPA2-PSK-SHA256 security. */
 	WIFI_SECURITY_TYPE_PSK_SHA256,
-	/** WPA3-SAE security. */
-	WIFI_SECURITY_TYPE_SAE,
+	/** WPA3-SAE security with hunting-and-pecking loop. */
+	WIFI_SECURITY_TYPE_SAE_HNP,
```

Comment:
To make it backwards compatible, you can do below
```suggestion
	WIFI_SECURITY_TYPE_SAE_HNP = WIFI_SECURITY_TYPE_SAE,
```

Reply:
Updated